src/app/docs/checkbox/checkbox-docs.component.html

```diff
@@ -0,0 +1,14 @@
+<sky-docs-demo-page
+  moduleName="SkyCheckboxModule"
+  moduleSourceCodePath="src/app/public/modules/checkbox/"
+  pageTitle="Checkbox"
+>
+  <sky-docs-demo-page-summary>
+    The checkbox component is a SKY UX-themed replacement for an HTML <sky-code>input</sky-code> element with <sky-code>type="checkbox"</sky-code>. When users select a checkbox, its value is driven through an <sky-code>ngModel</sky-code> attribute that you specify on the <sky-code>sky-checkbox</sky-code> element.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
```

Comment:
Should we put the accessibility content from https://developer.blackbaud.com/skyux/components/checkbox#accessibility on the Design tab for now? Or should we delete the `sky-docs-design-guidelines` elements to hide the Design tab for now? Alternately, we could leave it as is for now and have a "coming soon" message like on data manager.

Reply:
I added an Accessibility section and included that info, but I didn't add the rest of the design docs.